src/main.js

```diff
@@ -126,12 +126,28 @@ function deleteSavedCover() {
 
 function saveCustomBook() {
   event.preventDefault();
-  if (inputCover.value && inputTitle.value && inputFirstDescriptor.value && inputSecondDescriptor.value) {
+  if (validateForm()) {
     createNewBook();
     showNewBookHome();
   }
 }
 
+function validateForm() {
+  if (!inputCover.value) {
+    alert('Please enter a cover image URL!');
```

Comment:
There is a certain Turing teacher who is not a big fan of using "alerts". My advice would be to research other more efficient ways of form validation for future projects. You could also expand on this by making it more specific if you wanted to ensure user uses certain strings like ".com". Anyways, project looks great!